jira/client.py

```diff
@@ -4330,16 +4324,16 @@ def add_user(
             directoryId (int): The directory ID the new user should be a part of (Default: 1)
             password (Optional[str]): Optional, the password for the new user
             fullname (Optional[str]): Optional, the full name of the new user
-            notify (bool): Whether or not to send a notification to the new user. (Default: False)
-            active (bool): Whether or not to make the new user active upon creation. (Default: True)
-            ignore_existing (bool): Whether or not to ignore and existing user. (Default: False)
-            applicationKeys (Optional[list]): Keys of products user should have access to
+            notify (bool): Whether to send a notification to the new user. (Default: False)
+            active (bool): Whether to make the new user active upon creation. (Default: True)
+            ignore_existing (bool): Whether to ignore and existing user. (Default: False)
```

Comment:
(non-blocking), I find it clearer to be explicit on what True does for boolean arguments.

e.g. 
```
notify (bool): True will send a notification to the new user. (Default: False)
```

Reply:
It's a personal preference I share... Though I would prefer to open a new PR and apply this consistently amongst all doc strings.

Reply:
https://github.com/pycontribs/jira/tree/unify_docstrings_for_bool_to_refer_whats_true